enterprise/server/workflow/config/config.go

```diff
@@ -144,6 +148,21 @@ func NewConfig(r io.Reader) (*BuildBuddyConfig, error) {
 	return cfg, nil
 }
 
+func KytheIndexingAction(targetRepoDefaultBranch string) *Action {
+	var pushTriggerBranches []string
+	if targetRepoDefaultBranch != "" {
+		pushTriggerBranches = append(pushTriggerBranches, targetRepoDefaultBranch)
+	}
+	return &Action{
+		Name: KytheActionName,
+		Triggers: &Triggers{
+			Push: &PushTrigger{Branches: pushTriggerBranches},
+		},
+		// Note: default Bazel flags are written by the runner to ~/.bazelrc
+		BazelCommands: []string{"build --override_repository kythe_release=$KYTHE_DIR //..."},
```

Comment:
do you want to add --remote_cache here too? (bes_backend is enabled by default via our generated bazelrc, but remote_cache isn't)

Reply:
Done.